src/main/java/com/bullhornsdk/data/api/helper/RestJsonConverter.java

```diff
@@ -84,19 +85,19 @@ private ObjectMapper createObjectMapperWithRootUnWrap() {
     }
 
     public <T> T jsonToEntity(String jsonString, Class<T> type, ObjectMapper objectMapper) {
-        T entity = null;
 
         try {
-            entity = objectMapper.readValue(jsonString, type);
+            T entity = objectMapper.readValue(jsonString, type);
+            return entity;
         } catch (JsonParseException e) {
             log.error("Error parsing jsonString to " + type + ". jsonString = " + jsonString, e);
```

Comment:
done. split the catch up as the the exceptions were child classes of IOException.